metabase/metabase
# ci(sdk): Remove unnecessary uberjar process from SDK workflow

Closes https://github.com/metabase/metabase/issues/48728

### Description

This PR removes the unnecessary uberjar process from the SDK workflow. We don't need it anymore. You can read the issue for more details.

### How to verify
I tested with this commit https://github.com/metabase/metabase/pull/48729/commits/f9caf23d00ff69b660f9e6830b47603c204d5a67.

- View the [SDK release run detail](https://github.com/metabase/metabase/actions/runs/11345728654) (using `npm publish --dry-run` so we don't actually publish anything). It should be green.
- The Docs PR [Update SDK version to 0.1.39-test1](https://github.com/metabase/metabase/pull/48730#top) is created successfully.
